jeeves/common.py

```diff
@@ -79,6 +79,8 @@ def generate_html_file(htmlcode, remind=False):
 def percent(part, whole):
 	''' basic percent function
 	'''
+	if whole == 0:
+		return 0
```

Comment:
What case were you hitting where this occurred, out of curiosity?

Reply:
It occurs when you use filter. You end-up with no jobs for specific osp version if you are lucky.

Reply:
![percent](https://user-images.githubusercontent.com/20891964/116442614-fcfa9f80-a852-11eb-9865-bf37e60f1e67.png)

Reply:
:+1: